hetnet/neo4j/deployment/README.md

```diff
@@ -0,0 +1,34 @@
+# Neo4j Deployment
+
+This directory includes files to deploy neo4j-hetionet on Ubuntu 18.04 box.
+
+## Prerequisites of Deployment Box:
+ - OS: Ubuntu 18.04 or later
+ - User account `ubuntu` that has `sudo` privilege
+
+## Deployment Steps:
+
+Open `install_ssl.sh` and set the following two lines correctly:
+```shell
+EMAIL="foo@bar.org"                 # set email address associated with SSL certificate
+SSL_DOMAIN="dhu-foo.greenelab.com"  # set your SSL domain name
```

Comment:
Perhaps we should just say "edit the values for `EMAIL` and `SSL_DOMAIN`. I don't want someone to accidentally edit them here and think that will have an effect